testsuite/tests/backtrace/tailcall.ml

```diff
@@ -0,0 +1,10 @@
+let g () = raise Not_found
+
+let h () =
+  (* "g ()" is inlined by native compiler, but the location of the call should
+     be dropped from the backtrace as it is in tail position. *)
+  g ()
```

Comment:
Also to expose potential complexity for the flambda version, you may make 2 versions. One with h also inlined and one where it isn"t.